.github/workflows/release.yml

```diff
@@ -27,3 +27,16 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           AUR_SSH_KEY: ${{ secrets.AUR_SSH_KEY }}
           HOMEBREW_TAP_GITHUB_TOKEN: ${{ secrets.HOMEBREW_TAP_GITHUB_TOKEN }}
+
+      - name: Generate release tarball
+        run: |
+          VERSION=${GITHUB_REF_NAME#v}
```

Comment:
Can you explain the syntax?

Reply:
`${GITHUB_REF_NAME#v}` removes "v" prefix from version for convenience